packages/common/src/lib/entity/entity-table/entity-table.component.ts

```diff
@@ -522,13 +522,13 @@ export class EntityTableComponent implements OnInit, OnChanges, OnDestroy {
    * @internal
    */
   getTrackByFunction() {
```

Comment:
Angular offre un type pour typer la méthode, on pourrait l'utiliser
```suggestion
  getTrackByFunction(): TrackByFunction<RowData> {
    return (_index, row) => row.record.ref;
  }
```